Backend/BackendAPI/Repository/Repositories/RentalRepository.cs

```diff
@@ -55,5 +61,15 @@ public override Rental Update(Rental component)
             dbContext.Entry(GetByID(component.ID)).CurrentValues.SetValues(component);
             return component;
         }
+
+        public IEnumerable<Rental> FindActiveRentals(int userId)
+        {
+            return dbContext.Rentals
```

Comment:
wydaje mi się, że tutaj można zgodnie z pozostałymi metodami użyć GetAllRentals (oczywiście to drobnostka, więc jak ci się nie chce poprawiać to niech zostanie)